### ISSUE TITLE
Memorable domain name

### ISSUE THREAD

**Issue by nvkelso** [iandees/wof-editor#33]
instead of the crazy Lambda domain name

**Comment by iandees** [comment]
Useful docs here: https://serverless.com/blog/serverless-api-gateway-domain/

**Comment by nvkelso** [comment]
writefield.nextzen.org

**Comment by iandees** [comment]
https://writefield.nextzen.org/ works now.

For posterity: I tried using CloudFront in front of the regional API Gateway endpoint and CNAME'd the domain to that CloudFront domain. This didn't work, though, because CloudFront would cache responses and hide the error messages that came out of the web app. Preventing caching on CloudFront worked, but the login system wouldn't work – I think CloudFront wasn't forwarding cookie headers?

I ended up removing the CloudFront distribution and added the custom domain straight to API Gateway, which seems to do the expected.
